packages/components/psammead-caption/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.1.24| [PR#4630](https://github.com/bbc/psammead/pull/4630) Update text colour to C_GREY_6 |
```

Comment:
Think this should be `4.1.25` :)

Reply:
Copy and Paste, fast but risky 😂. I fixed this, thanks